packages/radix-vue/src/Select/SelectGroup.vue

```diff
@@ -0,0 +1,73 @@
+<script lang="ts">
+import type { Ref, InjectionKey } from "vue";
+import type { DataOrientation, Direction } from "../shared/types";
+
+type TypeEnum = "single" | "multiple";
+
+export interface SelectGroupProps {
+  type?: TypeEnum;
+  value?: string;
+  defaultValue?: string;
+  disabled?: boolean;
+  rovingFocus?: boolean;
+  orientation?: DataOrientation;
+  dir?: Direction;
+  loop?: boolean;
+  modelValue?: string | string[];
+}
+
+export const SELECT_GROUP_INJECTION_KEY =
+  Symbol() as InjectionKey<SelectGroupProvideValue>;
+
+export interface SelectGroupProvideValue {
+  type: TypeEnum;
+  modelValue?: Readonly<Ref<string | string[] | undefined>>;
+  changeModelValue: (value: string) => void;
+  parentElement: Ref<HTMLElement | undefined>;
+}
+</script>
+
+<script setup lang="ts">
+import { toRef, provide } from "vue";
+import { PrimitiveDiv, usePrimitiveElement } from "@/Primitive";
+
+const props = withDefaults(defineProps<SelectGroupProps>(), {
+  type: "single",
+});
+
+const emits = defineEmits(["update:modelValue"]);
+
+const { primitiveElement, currentElement: parentElement } =
+  usePrimitiveElement();
+
+provide<SelectGroupProvideValue>(SELECT_GROUP_INJECTION_KEY, {
+  type: props.type,
+  modelValue: toRef(() => props.modelValue),
```

Comment:
perhaps we could use `useVModel` here?

Reply:
sure, there's a ton of refactoring needed here, especially to support multiple select, I will and do tonight. This especially, we dont actually need a v-model in the group since no value are gonna passed to the group, this is just because I reused from the previous dropdownmenu component

Reply:
@khairulhaaziq  is there a multiple select features for Select component?

Reply:
> @khairulhaaziq is there a multiple select features for Select component?

Ouh just saw radix doesnt have it, we may extend it, I really like how easy to do multiple select with headless UI, API can be similar to radix’s accordion

Reply:
True. I think this could be an enhancement in coming day